pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -206,11 +212,26 @@ func (f *FakeDockerClient) StartContainer(id string, hostConfig *docker.HostConf
 			Config:     &docker.Config{Image: "testimage"},
 			HostConfig: hostConfig,
 			State: docker.State{
-				Running: true,
-				Pid:     os.Getpid(),
+				Running:   true,
+				Pid:       os.Getpid(),
+				StartedAt: time.Now(),
 			},
 			NetworkSettings: &docker.NetworkSettings{IPAddress: "1.2.3.4"},
 		}
+		if f.ContainerMap != nil {
```

Comment:
I'd rather not - it's sooo deeply intertwined into testing infrastructure, that it's hard. (Read as: I spent half an hour trying to figure out how it's passed to the kubelet_test and I really don't think it's worth spending more)